pom.xml

```diff
@@ -271,7 +271,7 @@
         <dependency>
             <groupId>com.github.ibi-group</groupId>
             <artifactId>gtfs-lib</artifactId>
-            <version>2bd924c6e5f5244a279c9d94298c14c0b3d47b35</version>
+            <version>26824f52fd7f8e74069ebfaa197754cefb6e5a03</version>
```

Comment:
(Remember to update GTFS-lib version number.)

Reply:
Fixed in: https://github.com/ibi-group/datatools-server/pull/599/commits/10d924c72b5e18346ba2146ddb90601d35eda486